.github/workflows/contracts.yml

```diff
@@ -101,6 +101,7 @@ jobs:
     if: |
       github.ref == 'refs/heads/master'
         && github.event_name != 'pull_request'
+        || github.ref == 'refs/heads/rfc-18/load-env-var'
```

Comment:
Can we remove it?

Reply:
Done, see 54ef99401b67476551e4e023357de64df9604501.